src/ray/common/ray_config_def.h

```diff
@@ -128,6 +128,11 @@ RAY_CONFIG(int64_t, worker_cap_max_backoff_delay_ms, 1000 * 10)
 /// even balancing of load. Low values (min 0.0) encourage more load spreading.
 RAY_CONFIG(float, scheduler_spread_threshold, 0.5);
 
+/// Whether to only report the usage of primary copies of objects in the
+/// object_store_memory resource. This means nodes holding secondary copies only
+/// will become eligible for removal in the autoscaler.
+RAY_CONFIG(bool, scheduler_report_primary_copy_bytes_only, true)
```

Comment:
technically the name is misleading because secondary copies that are pinned are still reported?